### PR TITLE
feat(table): Table component

### PR DESCRIPTION
**Pull request by justwiner**
- [ ] Component Migration

### CODE REVIEW

File: packages/utils/util.ts [hunk 1]
```diff
@@ -156,3 +156,18 @@ export function getRandomInt(max: number) {
 }
 
 export { isVNode } from 'vue'
+
+// TODO: use native Array.find, Array.findIndex when IE support is dropped
```
Comment (zazzaz) [inline packages/utils/util.ts:160]: Just use Array.find, please

**Comment by zazzaz** [comment]
Cool~

@justwiner Just a reminder that it's better to rebase the origin/dev branch daily to reduce conflicts.

File: packages/utils/util.ts (hunk 1 above)
Comment (justwiner) [reply packages/utils/util.ts:160]: Ok!
Optimization will be carried out later, including type checking

File: packages/table/src/store/helper.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import Store from './index'
-import debounce from 'throttle-debounce/debounce'
```
Comment (zazzaz) [inline packages/table/src/store/helper.ts]: please use `lodash` instead, package throttle-debounce will be removed later

**Pull request closed without merging**